src/redash_stmo/datasource_link/bundle/datasource_link.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { react2angular } from 'react2angular';
+
+class DatasourceLink extends React.Component {
+  static propTypes = {
+    clientConfig: PropTypes.object.isRequired,
+    datasourceId: PropTypes.number.isRequired,
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      type_name: '',
+      doc_url: '',
+    };
+  }
+
+  _loadURLData() {
```

Comment:
Any reason to have this prefixed with an underscore?

Reply:
This was just to indicate it's not a built-in React function like the rest. I can remove it if it's more confusing than useful. It doesn't necessarily seem to be a common practice, I've just seen it in a couple of places.

Reply:
I don't have a strong opinion either way, just noticed and wondered about the best practice in modern JS is.